### DISCUSSION TITLE
New Relic Monitoring

### DISCUSSION THREAD

**Discussion by matheussss1** [adonisjs/core#4237] [Help]
Hello, my name is Matheus and I am currently a FullStack developer. Two years ago, my development team chose AdonisJS as the server-side framework for our application. Fast forward two years, and we have become a renowned startup, winning multiple awards and serving over 500 active clients. With all this growth, we felt the need to analyze the performance of our application. That's when we turned to New Relic and installed their monitoring tool.

The reason I am requesting this feature is because the framework currently lacks any hints or possibilities for integrating with New Relic. As a result, our requests are uncategorized (without any reference to the controller), among other issues. I am reaching out for your help, and if you're interested, I would be more than willing to share how we have been using your framework in our startup."

**Comment by thetutlage** [comment]
Hello @matheussss1 

It's great to hear that you and your team have been using AdonisJS for a long time and are able to build a great company around it.

You are looking for an integration deeply integrated with New-Relic or exposes data in an open format like Opentelemetry.

Creating an integration for this is a time-consuming task; hence, you and your team have to decide if you want to invest that time. However, you can always create discussions to talk about specific issues that help you move forward in building this integration.

**Comment by OneCricketeer** [comment]
Related discussion on opentelemetry - https://github.com/adonisjs/core/issues/4138
